src/select.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import JWSelect from 'react-select';
+import classNames from 'classnames';
+import identity from './helpers/identity';
+
+export default class Select extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: props.defaultValue
```

Comment:
we should modify this to `value: props.value || props.defaultValue` or something like that. If the user wants a controlled input (via supplying `value) and calls `getValue` right away they will get undefined.